ts/components/services/NewServiceListItem.tsx

```diff
@@ -55,6 +55,7 @@ const NewServiceListItem = (props: Props): React.ReactElement => {
 
   return (
     <ListItemComponent
+      accessibilityRole={"link"}
```

Comment:
Since it is a list item, could it a `button` instead?
```suggestion
      accessibilityRole={"button"}
```